packages/bpk-component-carousel/src/BpkCarousel.scss

```diff
@@ -0,0 +1,47 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2016-2019 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+@import '~bpk-mixins/index';
+
+.bpk-carousel {
+  display: flex;
+  justify-content: space-between;
+
+  &__wrapper {
+    overflow: hidden;
+  }
+
+  &__container {
+    display: flex;
+    justify-content: space-between;
+    flex: 1 0;
+    transition: margin-left $bpk-duration-base;
```

Comment:
Have we tested this in IE11? If I remember correctly transition + calc does not work :/.
Also a suggestion, maybe not for this iteration but using transition like this is not very performant, translateX would give a smother result.

Reply:
Yes has been verified on IE and Edge, works as expected 😄

Reply:
It has been tried in IE and the only problem is that it doesn't do the smooth animation which might not be awful but something to look at for the future

Reply:
👍